### PR TITLE
Task/add warning for external storage

### PR DESCRIPTION
**Pull request by bvered**

### CODE REVIEW

**Comment by bvered** [comment]
[add warning msg for system calac & pool calc in case external storage is in-use](https://trello.com/c/abtU3szu/261-add-warning-msg-for-system-calac-pool-calc-in-case-external-storage-is-in-use)

File: src/components/capacity-card/generic-raw-capacity-card/generic-raw-capacity-card.tsx [hunk 1]
```diff
@@ -73,9 +74,13 @@ export const RawCapacityCard: React.FC<RawCapacityCardProps> = (props) => {
     const invalidStats = totalCapacity.value == INVALID_PROMETHEUS_CHILD_STATS ||
         usedCapacity.value == INVALID_PROMETHEUS_CHILD_STATS || availableCapacity.value == INVALID_PROMETHEUS_CHILD_STATS
     loadError = loadError || invalidStats || invalidValue
-
     const errorMessage:string = invalidStats? t('Physical capacity overview is unsupported for child pools.'): t('Not available')
 
+    if ( internalStorageCount > 0  || invalidStats) {
+        showExternalStorageWarning = false
+    }
+    const warningMessage:string = showExternalStorageWarning? t("* " + 'Statistics might have discrepancy with FS UI'): t('Not available')
```
[the remark this answers is not in the document]
Comment (bvered) [reply src/components/capacity-card/generic-raw-capacity-card/generic-raw-capacity-card.tsx]: fixed.

File: src/components/capacity-card/pool-physical-raw-capacity-card/pool-physical-raw-capacity-card.tsx [hunk 2]
```diff
@@ -52,6 +54,13 @@ export const PoolPhysicalRawCapacityCard: React.FC<PoolRawCapacityCardProps> = (
     const loadError = totalCapacityLoadError || usedCapacityLoadError || availableCapacityLoadError
     const loading = totalCapacityLoading || usedCapacityLoading || availableCapacityLoading
 
+    const [internalStorage, , ] = useCustomPrometheusPoll({
+        query: FlASHSYSTEM_POOL_QUERIES(name, pool_name, StorageDashboardQuery.PoolIsInternalStorage),
+        endpoint: "api/v1/query" as any,
+        samples: 60,
+    });
+    const internalStorageCount = _.get(internalStorage, "data.result[0].value[1]");
```
[the remark this answers is not in the document]
Comment (bvered) [reply src/components/capacity-card/pool-physical-raw-capacity-card/pool-physical-raw-capacity-card.tsx:62]: https://trello.com/c/d7KH62CK/270-improve-console-prometheus-result-parsing-combine-into-one-file

**Review by tyichye19 — APPROVED** [review]
LGTM

**Pull request merged**